util/headerreader/header_reader.go

```diff
@@ -201,6 +213,22 @@ func (s *HeaderReader) possiblyBroadcast(h *types.Header) {
 	}
 }
 
+func (s *HeaderReader) getPendingCallBlockNumber() (*big.Int, error) {
+	if s.isParentChainArbitrum {
+		arbSys, err := precompilesgen.NewArbSys(types.ArbSysAddress, s.client)
+		if err != nil {
+			return nil, err
+		}
+		blockNumber, err := arbSys.ArbBlockNumber(&bind.CallOpts{Context: s.GetContext(), Pending: true})
+		if err == nil {
+			return blockNumber, nil
+		} else {
+			log.Trace("failed to get pending block number with ArbSys, trying the normal EVM way", "err", err)
```

Comment:
Now that we know for certain that this is an arbitrum chain, we should just return the error in this case.

Reply:
done.